projects/fsastorefrontlib/src/cms-components/my-account/address-book/address-form/address-form.component.html

```diff
@@ -156,9 +154,9 @@
       </div>
     </div>
   </div>
-  <div class="cx-checkout-btns row">
+  <div class="cx-checkout-btns row w-100">
```

Comment:
w-100 doesn't seem right here. Please reference Personal details or Email address pages for entire HTML structure change.